python/cudf_polars/cudf_polars/containers/dataframe.py

```diff
@@ -277,7 +277,7 @@ def discard_columns(self, names: Set[str]) -> Self:
         """Drop columns by name."""
         return type(self)(column for column in self.columns if column.name not in names)
 
-    def select(self, names: Sequence[str]) -> Self:
+    def select(self, names: Sequence[str] | Mapping[str, Any]) -> Self:
```

Comment:
Maybe change `Sequence[str]` to `Iterable[str]` instead.

Reply:
Will tack this on another PR